### PR TITLE
Removing bpg commands to resolve circular dependency

### PR DESCRIPTION
**Pull request by dgsudharsan**
### Description of PR

Summary:
Fixes # (issue)
This PR should be followed by https://github.com/sonic-net/sonic-mgmt/pull/15466.

There is a circular dependency where the sonic-utilities submodule after this change https://github.com/sonic-net/sonic-utilities/pull/3605 will not get update unless show techsupport test pass. However the test cannot pass until we have sonic-utilities submodule updated.

To overcome this create a temporary PR to remove the techsupport commands that create this dependency. After the submodule update is done  https://github.com/sonic-net/sonic-mgmt/pull/15466 can be merged
### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [ ] 202405

### CODE REVIEW

**Review by xwjiang-ms — APPROVED** [review]
LGTM, please remember to revert this PR after change got merged

**Comment by dgsudharsan** [comment]
@xwjiang-ms @yxieca  Can we merge this to unblock sonic-utilities pipeline checker? This passed all checkers except for the one listed as optional.

**Review by wangxin — APPROVED** [review]

**Pull request merged**
